src/AWS.Deploy.CLI/Commands/DeployCommand.cs

```diff
@@ -706,6 +706,10 @@ private async Task CreateDeploymentBundle(Orchestrator orchestrator, Recommendat
                     }
                     else
                     {
+                        if (!selectedRecommendation.ProjectDefinition.HasDockerFile)
+                        {
+                            throw new FailedToCreateDeploymentBundleException(DeployToolErrorCode.FailedToCreateContainerDeploymentBundleFromGeneratedDockerFile, "Failed to create a deployment bundle from generated Docker file");

```

Comment:
For error message what about something like to give the user some direction on what to do to fix the issue.
```
Failed to create a container image from generated Docker file. Please edit the Dockerfile at {DockerfilePath} to correct the required build steps for the project. Common errors are missing project dependencies not included in the Dockerfile.
```

Reply:
Added the error message.